tests/bpftool_tests/bpftool_tests.cpp

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Microsoft Corporation
```

Comment:
Add this test to CI/CD?

Reply:
The PR description already says "A future PR will add it to the KM test runner in CI/CD." :)
This is part 1 of a set.

Reply:
OK, great. We should now be able to capture and upload code coverage from a self-hosted runner as well.